packages/block-library/src/index.js

```diff
@@ -122,16 +124,23 @@ const registerBlock = ( block ) => {
 };
 
 /**
- * Function to get all the core blocks in an array.
- *
- * @example
- * ```js
- * import { __experimentalGetCoreBlocks } from '@wordpress/block-library';
- *
- * const coreBlocks = __experimentalGetCoreBlocks();
- * ```
+ * Function to get all the block-library blocks in an array
  */
-export const __experimentalGetCoreBlocks = () => [
+const getAllBlocks = () => [
+	// The blocks listed here are wrapped in a conditional expression
+	// by the block-library/babel-plugin.
```

Comment:
Nit:
```suggestion
	// by the `packages/block-library/babel-plugin.js`.
```

Could be also "// by the `@wordpress/block-library/babel-plugin` Babel plugin."

Overall, this comment is excellent. It greatly explains what happens behind the scenes ❤️

Reply:
Suggestion committed.

Aw, thanks man!